src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
So on the second note, what that would end up with is importing a collection of utilities (I think we recently said we wanted `Lib` -> `Util`). So, it would look vaguely like this (but with better names maybe):

```js
import { dom, browserCheck, kittens, puppies, bryanCranston } from './util';

browserCheck.IS_IE8;
```